game/core/constants.py

```diff
@@ -41,7 +41,7 @@
 
 ITEM_MAP = {"pickaxe": 43}
 
-SAVE_FILE_DIR = "src/resources/__cache__"
+SAVE_FILE_DIR = "game/assets/maps"
 PLAYER_SAVE_FILE = f"{SAVE_FILE_DIR}/player_save_file"
 MAP_SAVE_FILE = f"{SAVE_FILE_DIR}/map_save_file.json"
```

Comment:
This could be paths by default so that you don't need to convert to Path later

```suggestion
SAVE_FILE_DIR = Path("game/assets/maps")
PLAYER_SAVE_FILE = SAVE_FILE_DIR / "player_save_file"
MAP_SAVE_FILE = SAVE_FILE_DIR / "map_save_file.json"
```

Reply:
Good idea! I pushed up a separate commit for it since this meant I needed to change `game_state.py` as well